README.md

```diff
@@ -1,27 +1,86 @@
-# Forks and Clones
+# Forking and Cloning Github Repositories
+
+## Problem Statement
+
+If you are familiar with how to create local repositories, you have the ability to
+create a logged history of your project. What's great about open source is that lots
+of people are creating new repositories all the time and sharing their work. You can
+take your repository and push it to a remote to share it, or you can clone a repository
+to your local machine from a remote and make a copy on your own system. Due to permissions
```

Comment:
Let's not muddy things this early for a beginner. Cut from Due to permissions to end of paragraph and let's address why fork after we teach `clone`.